Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml

```diff
@@ -24,8 +24,8 @@
 		<SourcePathVariable Id="SourceModelPlugins" Value="%SourceBin%\plugins\Tekla\Model\%ExtensionName%"/>
 		<SourcePathVariable Id="SourceDrawingApps" Value="%SourceBin%\applications\Tekla\Drawings\%ExtensionName%"/>
 		<SourcePathVariable Id="SourceModelApps" Value="%SourceBin%\applications\Tekla\Model\%ExtensionName%"/>
-		<SourcePathVariable Id="SourceCommmon" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\common"/>
-		<SourcePathVariable Id="SourceUsaRoot" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\USA"/>
+		<!-- <SourcePathVariable Id="SourceCommmon" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\common"/> -->
+		<!-- <SourcePathVariable Id="SourceUsaRoot" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\USA"/> -->
```

Comment:
Why are these now turned off?

Reply:
the folders are empty or non existent in the repo